superset/views.py

```diff
@@ -1104,6 +1104,38 @@ def ping():
     return "OK"
 
 
+class KV(BaseSupersetView):
+
+    """Used for storing and retrieving key value pairs"""
+
+    @log_this
+    @expose("/store/", methods=['POST', 'GET'])
+    def store(self):
+        value = request.form.get('data')
+        if not value:
+            return Response("Error: data can not be empty", status=404)
+        base_url = request.form.get('baseUrl') \
+            if request.form.get('baseUrl') else ''
+        obj = models.KeyValue(value=value)
+        db.session.add(obj)
+        db.session.commit()
+        return("http://{request.headers[Host]}/{base_url}?id={obj.id}".format(
+            request=request, base_url=base_url, obj=obj))
+
+    @log_this
+    @expose("/<key_id>/", methods=['GET'])
+    def get_value(self, key_id):
+        if key_id == 'null':
+            return Response("Error: Value cannot be retrieved", status=404)
+        kv = db.session.query(models.KeyValue).filter_by(id=key_id).first()
```

Comment:
for one() it won't return None when entry is not retrieved, first() returns None when it can't be retrieved, like we use in a lot of other queries in views.py